src/main/java/com/uber/cadence/worker/Worker.java

```diff
@@ -528,28 +597,49 @@ private String getStickyTaskListName() {
       private PollerOptions stickyWorkflowPollerOptions;
       private Scope metricScope;
 
+      /**
+       * When set to true it will create an affinity between the worker and the workflow run it's
+       * processing. Workers will cache workflows and will handle all decisions for that workflow
+       * instance until it's complete or evicted from the cache.
+       */
       public Builder setEnableStickyExecution(boolean enableStickyExecution) {
```

Comment:
Let's change it to setDisableStickyExecution as the default value of a boolean property is false and we want sticky on by default.

Reply:
Will do this in a separate CR before release